packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -652,16 +652,16 @@ export const DatePickerMixin = (subclass) =>
     }
 
     /**
-     * Select date on user interaction and set the flag
-     * to fire change event if necessary.
+     * Sets the given date as the value and fires a change event
+     * if the value has changed.
      *
-     * @param {Date} dateToSelect
-     * @protected
+     * @param {Date} date
+     * @private
      */
-    _selectDate(dateToSelect) {
```

Comment:
**note:** I haven't found any reason to keep this method protected. It doesn't seem to be used anywhere outside.